src/core/helpers.js

```diff
@@ -1252,39 +1252,61 @@ wb.escapeAttribute = function( str ) {
 
 /*
 * Find most common Personal Identifiable Information (PII) in a string and return either the cleaned string either true/false
-* @param {string} str
-* @param {boolean} toClean
+* @param{string} str (required)- the content that needs to be verified
+* @param {boolean} toClean - if true will scrub the content
+* @param {object} the 2nd param can also be an  object having the following properties:*
+* 	{boolean} bMarker (optional) - if true, will replace the scrubbed values with string "[REDACTED/CAVIARDÉ]"
+* 	{string} caseOnly (optional) - this param should match a key name from the "regEx" object. If present, it will search and replace only the values corresponding that pattern
+* 	{regex} customCase (optional) - this param is a regex. It will search and replace the values corresponding that pattern
+
 * @return {string | true | false}
 * @example
 * wb.findPotentialPII( "email:test@test.com, phone:123 123 1234", true )
 * returns "email:, phone:",
+*
 * wb.findPotentialPII( "email:test@test.com, phone:123 123 1234", false )
 * returns true
+*
+* wb.findPotentialPII( "email:test@test.com, phone:123 123 1234", { "bMarker":true, "caseOnly":"email" } )
+* returns "email:[REDACTED/CAVIARDÉ], phone:123 123 1234"
+*
+* wb.findPotentialPII( "email:test@test.com, phone:123 123 1234, numéro de cas 12345678", { "bMarker":true, "customCase":/\b(?:case[\s-]?number[\s\-\\.]?(?:\d{5,10}))|(?:numéro[\s-]?de[\s-]?cas[\s\-\\.]?(?:\d{5,10}))/ig })
+* returns "email:[REDACTED/CAVIARDÉ], phone:[REDACTED/CAVIARDÉ], [REDACTED/CAVIARDÉ]"
 */
-wb.findPotentialPII = function( str, toClean ) {
-
-	if ( typeof str  !== "string" ) {
+wb.findPotentialPII = function( str, opts ) {
```

Comment:
```suggestion
wb.findPotentialPII = function( str, scope, opts ) {
```
where:
* str: string to find Potential PII
* scope: Object or list to run the find to detect PII (if False => we use all current regex)
* opts: Option of hte PII transformation: toClean replace with marker, replace with one phrase.....

3e param

```
{
   markerText = FullBlock character,
   repeatMarker = true | false,
   allRuleDefaultIncluded = true | false,
   allRuleExcluded = true | false // default true,
}

var settings = $.extend( {}, defaultOpts, opts );
// object.assign

```